sui/deploy-gateway.js

```diff
@@ -87,6 +85,7 @@ async function processCommand(config, chain, options) {
             tx.pure.address(operator),
             separator,
             tx.pure(minimumRotationDelay),
+            tx.pure(options.previousSigners),
```

Comment:
deployment needs to support specifying upgrade policy, created a [task](https://axelarnetwork.atlassian.net/browse/AXE-4489)